nixos/modules/services/web-apps/wordpress.nix

```diff
@@ -268,10 +262,7 @@ in
   options = {
     services.wordpress = mkOption {
       type = types.submodule {
```

Comment:
I think the submodule is not needed anymore, I had to use it for `freeformType`, but it's not needed anymore.

We just need:

 ```nix
services.wordpress = {
  sites = mkOption { ... };
  webserver = mkOption { ... };
};